README.md

```diff
@@ -1,4 +1,189 @@
-Astarte Dashboard
-=================
 
-Astarte Dashboard allows users perform administrative tasks such as installing interfaces and triggers.
+[![Astarte logo][astarte-logo]](https://github.com/astarte-platform/astarte-dashboard)
+
+# Astarte Dashboard
+
+## Table of Contents
+
+- [About the project](#about-the-project)
+- [Usage](#usage)
+  - [Prerequisites](#prerequisites)
+  - [Configuration](#configuration)
+  - [Run it](#run-it)
+- [Contributing](#contributing)
+  - [Dependencies](#dependencies)
+  - [Starting up a local server](#starting-up-a-local-server)
+  - [Testing](#testing)
+- [License](#license)
+
+## About The Project
+
+[![Dashboard Home Screen Shot][dashboard-home-screenshot]](https://github.com/davidebriani/astarte/blob/docs/add-astarte-dashboard-documentation/doc/images/astarte-dashboard-realm-overview.png)
+
+The Astarte Dashboard is a web UI that allows users to analyze the status of
+their realm and perform administrative tasks such as install interfaces and triggers,
+manage devices, pipelines and flows on your Astarte realm.
```

Comment:
The Astarte Dashboard is a web UI that allows users to analyze the status of
their realm and **to** perform administrative tasks such as install**ing** interfaces and triggers,
manag**ing** devices, pipelines and flows on your Astarte realm.